ibis/pandas/execution.py

```diff
@@ -180,13 +180,21 @@ def execute_series_clip(op, data, lower, upper, scope=None):
 
 
 @execute_node.register(
-    ops.Quantile, pd.Series, float,
+    ops.Quantile, pd.Series, float
```

Comment:
You should be able to just do `(float, list)` here so you don't have to duplicate anything.